workflows/single_sample.nf

```diff
@@ -3,24 +3,13 @@ nextflow.preview.dsl=2
 //////////////////////////////////////////////////////
 //  Import sub-workflows from the modules:
 
+// Utils
 include '../src/utils/processes/utils.nf' params(params)
-
-include QC_FILTER from '../src/scanpy/workflows/qc_filter.nf' params(params)
-include NORMALIZE_TRANSFORM from '../src/scanpy/workflows/normalize_transform.nf' params(params)
-include HVG_SELECTION from '../src/scanpy/workflows/hvg_selection.nf' params(params)
-include SC__SCANPY__REGRESS_OUT from '../src/scanpy/processes/regress_out.nf' params(params)
-include NEIGHBORHOOD_GRAPH from '../src/scanpy/workflows/neighborhood_graph.nf' params(params)
-include DIM_REDUCTION_PCA from '../src/scanpy/workflows/dim_reduction_pca.nf' params(params)
-include '../src/scanpy/workflows/dim_reduction.nf' params(params)
-include '../src/scanpy/processes/cluster.nf' params(params)
-include CLUSTER_IDENTIFICATION from '../src/scanpy/workflows/cluster_identification.nf' params(params)
 include SC__H5AD_TO_FILTERED_LOOM from '../src/utils/processes/h5adToLoom.nf' params(params)
 include FILE_CONVERTER from '../src/utils/workflows/fileConverter.nf' params(params)
 
-// reporting:
-include UTILS__GENERATE_WORKFLOW_CONFIG_REPORT from '../src/utils/processes/reports.nf' params(params)
-include SC__SCANPY__MERGE_REPORTS from '../src/scanpy/processes/reports.nf' params(params)
-include SC__SCANPY__REPORT_TO_HTML from '../src/scanpy/processes/reports.nf' params(params)
+// Pipeline
+include single_sample as SCANPY_SINGLE_SAMPLE from '../src/scanpy/main.nf' params(params)
```

Comment:
I really don't like the fact that Scanpy now depends on the scrublet module. I thought we would move towards making the modules independent from each other and deal with links through workflows on the main repo?

Reply:
Ok so you would move the `single_sample_scrublet` pipeline in the `vsn-pipelines` ?
The thing is `scanpy` is also dependent on `pcacv` so it's tricky

Reply:
Removed `single_sample_scrublet` from `scanpy` repo to `vsn-pipelines`